Description: make build reproductive
 Needed when rebuilding distros
 .
 spectacle (0.25-1) unstable; urgency=low
 .
   * New upstream release.
   * debian: drop downstream patches
   * debian: install README* to docs
   * debian: Adopt package (Closes: #641955)
Author: Philippe Coval <rzr@gna.org>
Bug-Debian: https://bugs.debian.org/641955

---

Origin: https://github.com/rzr/spectacle.git
Bug-Debian: https://bugs.debian.org/775996

--- spectacle-0.25.orig/spectacle/dsc/Makefile
+++ spectacle-0.25/spectacle/dsc/Makefile
@@ -8,7 +8,7 @@ TEMPLATE_MODS=$(patsubst %.tmpl,%.py,$(T
 all: $(TEMPLATE_MODS)
 
 %.py: %.tmpl
-	$(CHEETAH) compile --settings='useStackFrames=False' $<
+	$(CHEETAH) compile --settings='useStackFrames=False,addTimestampsToCompilerOutput=False' $<
 	cp $@ __init__.py
 
 clean:
--- spectacle-0.25.orig/spectacle/spec/Makefile
+++ spectacle-0.25/spectacle/spec/Makefile
@@ -8,7 +8,7 @@ TEMPLATE_MODS=$(patsubst %.tmpl,%.py,$(T
 all: $(TEMPLATE_MODS)
 
 %.py: %.tmpl
-	$(CHEETAH) compile --settings='useStackFrames=False' $<
+	$(CHEETAH) compile --settings='useStackFrames=False,addTimestampsToCompilerOutput=False' $<
 	cp $@ __init__.py
 
 clean:
